en/SetupView/Joystick.md

```diff
@@ -30,12 +30,17 @@ To configure a joystick:
 
    ![Joystick setup - PlayStation](../../assets/setup/joystick_sony_playstation.jpg)
 1. Make sure your joystick is selected in the **Active joystick** dropdown.
-1. Press the **Calibrate** button and then follow the on-screen instructions to calibrate/move the sticks.
-1. Test the buttons and sticks work as intended by pressing them, and viewing the result in the Axis/Button monitor.
+1. Go to the **Calibrate** Tab, press the **Start** button and then follow the on-screen instructions to calibrate/move the sticks. The joystick is enabled as the last step of the calibration process.
+   ![Joystick setup - Calibration](../../assets/setup/joystick_calibration.jpg)
+1. Test the buttons and sticks work as intended by pressing them, and viewing the result in the Axis/Button monitor in the **General** tab.
 1. Select the flight modes/vehicle functions activated by each joystick button.
-1. Check the **Enable joystick input** checkbox to begin sending joystick commands to the vehicle.
```

Comment:
So joystick commands are now always/automatically sent when the joystick is connected?

Reply:
They are enabled as the last step of the calibration, check line 33.

Reply:
I see that now. It wasn't obvious. Do you think we should perhaps make that a note?

Reply:
Sure, feel free to add it!

Reply:
Done! This PR now just waiting on you @DonLakeFlyer

Reply:
In reality I haven't looked at Joystick config in ages. @dogmaphobic Was the one would did the latest rework of the ui. So he'd be better to review.

Reply:
It all seems fine with the exception of:

 **Enable Gimbal Control**: Enabled two additional channels for controlling a gimbal.

This is not enabled (internally there is no code to handle it). The joystick calibration and setup is too hardcoded for 4 attitude axis only. That needs some work before we can introduce more than 4 axis.

Reply:
@dogmaphobic Thanks. So that is what the UI shows - do we just add a note "not supported" or similar below that?

Reply:
There is a checkbox for something which you can check but does nothing?

Reply:
That's what it sounds like!